docs/components/connectors/custom-built-connectors/create-connector-from-rest.md

```diff
@@ -0,0 +1,114 @@
+
+1. Open the template editor for further customization:
+   - Select the newly created template from the template list.
+   - Click **Edit Template** to open the template editor.
+2. Customize the template:
+   - Add or remove fields as required.
+   - Adjust default values and input parameters.
+   - Update the description and other metadata as necessary.
+3. Save your changes to finalize the customization.
+
+:::note Warning
+If one of your variables is used somewhere else in your element template, be mindful that the variable using other variables is declared after.
+Otherwise, they will be considered `null`
+:::
+
+## Example Overview
+
+In this guide, you'll learn how to create a custom connector that fetches data from the Star Wars API (SWAPI). By following the steps outlined below, you'll be able to incorporate this connector into your BPMN workflows seamlessly.
+
+### Step-by-Step Instructions
+
+1. **Create a Rest outbound Connector task**
+
+   - Begin by creating a **Rest outbound connector** task in your workflow.
+   - Define the URL using a FEEL (Friendly Enough Expression Language) expression, utilizing the `resource` and `index` variables.
+     ![img.png](./img/custom-rest-connector.png)
+
+2. **Click on **Save as\*\*\*\*
+
+   - Save your current progress by selecting **Save as**. This will enable you to create a template from your configured connector.
+     ![img_1.png](./img/custom-save-as.png)
+
+3. **Fill Out Template Information**
+
+   - Provide essential details such as the **Template Name** and **Template Description**.
+   - These fields are crucial for identifying and understanding the purpose of the template.
+     ![img_2.png](./img/custom-save-as-template.png)
+
+4. **Open the New Connector Template in the Editor**
+
+   - Access the newly created template in the editor to make further customizations and configurations.
+
+5. **Hide Unwanted Properties**
+
+   - For properties that are not required in your connector, set their `type` to `Hidden`.
+     - Example: Since authentication is not necessary, set it to `Hidden`.
+       ![img.png](./img/custom-hide-authentication.png)
+   - Similarly, hide other fields such as `url`, `method`, `headers`, and `queryParameters`. If the `hidden` type does not apply, ensure that you remove the `feel` property.
+     ![img.png](./img/custom-hide-properties.png)
+
+6. **Create a Custom Group for the Star Wars API**
+
+   - Introduce a customized group named `swapi` for organizing your Star Wars-related properties.
+     ![img.png](./img/custom-create-new-group.png)
+
+7. **Define the Properties in the SWAPI Group**
+
+   - Map the properties within the new group to the variables defined earlier (`resource` and `index`).
+     - Set `resource` as a `Dropdown`.
+     - Set `index` as a `String`.
+       ![img.png](./img/custom-add-new-properties.png)
+
+8. **Add an Icon**
+
+   - Enhance your connector's visual appeal by adding an appropriate icon.
```

Comment:
The image referenced here and on line 106 seemed to be missing - but I am not sure an image for each step is required. The UI is fairly descriptive, and the code screenshots could be provided in codeblocks.